data_loading/loo_data_loader.py

```diff
@@ -13,10 +20,15 @@ def __init__(self, args):
         self.test = []
 
     def make(self, movie_to_entity_ratio=0.5, replace_movies_with_descriptive_entities=True, n_negative_samples=100,
-             keep_all_ratings=False, random_seed=42, movies_only=False, without_top_pop=False):
+             keep_all_ratings=False, random_seed=42, movies_only=False, without_top_pop=False,
+             position=0, filename="", progress=""):
         """
         Samples new positive and negative items for every user.
         """
+
+        # Pad to the longest possible filename because it looks nicer
+        filename = pad_to_length(f'[{filename}]', len('[./datasets/wtp-substituting-3-4/0.json]'))
```

Comment:
Do we have to hardcode this file name? :2nd_place_medal:

Reply:
Probably not but general fix is messy